README.md

```diff
@@ -21,6 +21,56 @@ microservices needing to check user authorization to operate on a resource can
 statelessly verify the user's authorization, making a request to arborist with
 the user's JWT and receiving a response for the authorization decision.
 
+## Setup
+
+### Quickstart
+
+You will need these:
+
+- [Go](https://golang.org/dl/)
+- [PostgreSQL](https://www.postgresql.org/download/)
+
+```bash
+go get -u github.com/uc-cdis/arborist
+make
+psql
```

Comment:
I'm a bit confused about the `psql` here

Reply:
uh oh, this part is not completed. will fix